molecule_common/base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos/systemd:latest
+
+RUN yum makecache fast && yum update -y && \
+    yum install -y python sudo yum-plugin-ovl && \
+    sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf
```

Comment:
I would suggest a more conservative regexp, `sed -i 's/^plugins=0$/plugins=1/g' /etc/yum.conf`, noting that sed will return 0 even if nothing is replaced.

If we'd want to fail the build if the replacement doesn't happen, then we can replace `sed` with `patch`, or `sed` + `grep`, or some other thing.